website/templates/website/documentation_page.html

```diff
@@ -1,6 +1,15 @@
 {% extends "website/base_template.html" %}
 {% load customfilters %}
 
+{% block extra_css %}
+<style type="text/css">
+	.section {
+    padding-top: 50px;
```

Comment:
Yep no trouble. Can you use something else than pixel sizing? em or relative sizing? This is really an open question, I'm so behind in CSS terms.